colossalai/inference/server/api_server.py

```diff
@@ -39,15 +38,16 @@
 app = FastAPI()
 
 
-# NOTE: (CjhHa1) models are still under development, need to be updated
-@app.get("/models")
-def get_available_models() -> Response:
-    return JSONResponse(supported_models_dict)
+@app.get("/ping")
```

Comment:
Is it possible to add an engine check API to check if the engine is running properly?

Reply:
added